tests/testthat/_snaps/select_best.md

```diff
@@ -16,15 +16,15 @@
     Code
       select_best(rcv_results, metric = "random")
     Condition
-      Error in `is_metric_maximize()`:
+      Error in `get_metric_direction()`:
```

Comment:
These messages ought to call out the user-called function rather than the checking function. I went ahead and updated snaps without making changes to the erroring code, as correcting this will require some more invasive changes to pass around calling environments that I'll implement in a separate PR.